test/NuGet.Core.FuncTests/NuGet.Packaging.FuncTest/SigningTests/TimestampTests.cs

```diff
@@ -47,7 +47,7 @@ public async Task Timestamp_Verify_WithOfflineRevocation_ReturnsCorrectFlagsAndL
                 using (var trustedServerRoot = TrustedTestCert.Create(
                     new X509Certificate2(rootCa.Certificate.GetEncoded()),
                     StoreName.Root,
-                    StoreLocation.LocalMachine))
+                    (RuntimeEnvironmentHelper.IsWindows || RuntimeEnvironmentHelper.IsMacOSX) ? StoreLocation.LocalMachine: StoreLocation.CurrentUser))
```

Comment:
again, not sure why

Reply:
The background is https://github.com/dotnet/runtime/blob/master/docs/design/features/cross-platform-cryptography.md#x509store, added in the comments. Thanks!